client/src/api/api.js

```diff
@@ -8,4 +8,12 @@ export const createUser = async (userData) => {
   return res.data
 }
 
+export const createCourse = async (applicationData) => {
+  const requestString = '/newApplication'
```

Comment:
Is this function for creating a new unit? If so, make sure to name it `createUnit` and set `requestString` to `'/language/course'`